static/css/_pages.css

```diff
@@ -444,3 +444,57 @@ a.oh-badge {
     margin-top: 25px;
   }
 }
+
+.top-buffer {
```

Comment:
The styles here have ignored that the way styles have been implemented in `_pages.css` was with page-specific scoping. Everything is nested within an id that starts with `#page-`, and this approach is being used to avoid styles that were only intended for a specific page (and might produce unintended consequences if the same class names are used elsewhere).

But because modals can be on any page, this approach probably should _not_ be used. New needed styles should be somewhere else, maybe `_buttons.css`.